selfapi.js

```diff
@@ -452,7 +449,10 @@ API.prototype = {
             html += header + ': ' + requestHeaders[header] + '\n';
           }
           if (request.body) {
-            html += '\n' + maybeJsonStringify(request.body).trim() + '\n';
+            if (Object.keys(requestHeaders) > 0) {
+              html += '\n';
```

Comment:
Good catch!

I use checks like these to add an empty line between meta-data like *exampleURL/status/headers* and example *body*, but in this case, since we always add an `exampleURL`, the empty line should always be there.

I'll re-read all export functions to make sure the check is correct (or absent) as needed.